core-web/libs/block-editor/src/lib/components/dot-block-editor/dot-block-editor.component.ts

```diff
@@ -90,7 +91,7 @@ export class DotBlockEditorComponent implements OnInit, OnDestroy {
 
     @Input() set value(content: Content) {
         if (typeof content === 'string') {
-            this.content = formatHTML(content);
+            this.content = formatHTML(this.getMarkdownHtml(content));
```

Comment:
What happen if the string is HTML?

Also... to prevent this block editor geeting bigger and bigger, can we use a library that we can load from a file and do all this transformation in the jsp?

Reply:
@fmontes If the string includes HTML the library is going to return that as HTML, I did the test for that.